apps/laboratory/tests/shared/pages/ModalPage.ts

```diff
@@ -263,9 +263,9 @@ export class ModalPage {
     await this.page.waitForTimeout(300)
   }
 
-  async updateEmail(mailsacApiKey: string, index: number) {
+  async updateEmail(mailsacApiKey: string) {
     const email = new Email(mailsacApiKey)
-    const newEmailAddress = email.getEmailAddressToUse(index)
+    const newEmailAddress = email.getEmailAddressToUse(1)
```

Comment:
@ignaciosantise I'd really appreciate it if you could tell more about the situation - why previously it had issues, how this is improving etc - in the PR description. Since tests have gotten overwhelming last few months and are very fragile , we might miss some details 🙏🏽

Reply:
Actually, this change is not needed. It was needed because Luka changed the `getEmailAddressToUse` function, but now apparently it changed again 😆 I'll revert this